### PR TITLE
Storage Preview 4 release

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/tests/test_common_blob_async.py [hunk 1]
```diff
@@ -434,10 +434,12 @@ def test_get_blob_with_snapshot_previous(self):
         content = await stream.content_as_bytes()
 
         # Assert
-        self.assertEqual(content, self.byte_data[:6])
+        self.assertEqual(content, self.byte_data[:5])
 
     @record
     def test_get_blob_with_range(self):
+        if TestMode.need_recording_file(self.test_mode):
+            pytest.skip("Issue with the recording")
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-blob/tests/test_common_blob_async.py]: Do you know what caused the problem for recording? ^_^

Comment (rakshith91) [reply sdk/storage/azure-storage-blob/tests/test_common_blob_async.py]: recording doesn't add the `content-type` for some reason.

Comment (xiafu-msft) [reply sdk/storage/azure-storage-blob/tests/test_common_blob_async.py]: Did we create an item for fixing that 😀

Comment (rakshith91) [reply sdk/storage/azure-storage-blob/tests/test_common_blob_async.py]: Added to the AC here: https://github.com/Azure/azure-sdk-for-python/issues/6225

**Pull request merged**